src/main/java/seedu/duke/command/GetCommand.java

```diff
@@ -25,11 +25,10 @@ public GetCommand(String name) {
     public void execute(ItemContainer list) throws ItemNotExistException {
         try {
             Item selectedItem = list.getItem(name);
-            String output = list.getDescription(selectedItem);
-            System.out.println(output);
+            System.out.printf((ItemContainer.ITEM_DESCRIPTION) + "%n",
+                selectedItem.getName(),selectedItem.getSellingPrice(), selectedItem.getPurchaseCost());
```

Comment:
It looks like it will be easier if there is a method to return such a string. Please create an issue as feature request, and assign to me. I will implement in the Item class in my next PR, then please create a new PR to replace this to use that method instead.